mne/viz/topomap.py

```diff
@@ -3456,11 +3456,9 @@ def _trigradient(x, y, z):
     """Take gradients of z on a mesh."""
     from matplotlib.tri import CubicTriInterpolator, Triangulation
 
-    with warnings.catch_warnings():  # catch matplotlib warnings
```

Comment:
I'm also trying some removals of `warnings.catch_warnings`, we'll see how many of them work :crossed_fingers: